DOCUMENTATION.md

```diff
@@ -385,21 +385,36 @@ E.g (rgba color) :
+|Classes (optional)|You can add css classes to stylize the image. Separate class with whitespace. Tranquilpeak integrate many css class to create nice effects: <ul><li><strong>fancybox</strong> : Generate a fancybox image.</li><li><strong>nocaption</strong> : Caption of the image will not be displayed.</li><li><strong>left</strong> : Image will float at the left.</li><li><strong>right</strong> : Image will float at the right.</li><li><strong>center</strong> : Image will be at center.</li><li><strong>fig-20</strong> : Image will take 20% of the width of post width and automatically float at left.</li><li><strong>fig-25</strong> : Image will take 25% of the width of post width and automatically float at left.</li><li><strong>fig-33</strong> : Image will take 33% of the width of post width and automatically float at left.</li><li><strong>fig-50</strong> : Image will take 50% of the width of post width and automatically float at left.</li><li><strong>fig-75</strong> : Image will take 75% of the width of post width and automatically float at left.</li><li><strong>fig-100</strong> : Image will take 100% of the width of post width.</li><li><strong>clear</strong> : Add a div with `clear:both;` style attached after the image to retrieve the normal flow of the post.</li></ul>|
 |Group (optional)| Name of a group, used to create a gallery. **Only for image with `fancybox` css class**|
 |Orignal image| Path to the original image.|
 |Thumbnail image (optional)| Path to the thumbnail image. If empty, the orignal image will be displayed.|
 |Width of thumbnail image (optional)| Width to the thumbnail image. If the thumbnail image is empty, width will be attached to thumbnail image created from original image. E.g : `150px` or `85%`.|
 |Height of thumbnail image (optional)| Height to the thumbnail image. If the thumbnail image is empty, height will be attached to thumbnail image created from original image. E.g : `300px` or `20%`.|
 |Title (optional)| Title of image displayed in a caption under image. `Alt` HTML attribute will use this title. E.g : `"A beautiful sunrise"`.|
  
+#### Video
+
+The video tag provides a way conveniently add videos. Take a look at some examples here: [Video Tag Demo](https://benmitchinson.com/Video-Tag-Demo/)
+
+Syntax : `{% video [classes] videoURL [Optional Poster (Thumbnail) URL] [Width] [Caption] %}`
+E.g : `{% video center https://sample-videos.com/video123/mp4/720/big_buck_bunny_720p_1mb.mp4 https://peach.blender.org/wp-content/uploads/title_anouncement.jpg 100% Poster (Thumbnail) Enabled %}`
```

Comment:
I suggest we replace `https://sample-videos.com/video123/mp4/720/big_buck_bunny_720p_1mb.mp4` with `https://example.com/video.mp4` and `https://peach.blender.org/wp-content/uploads/title_anouncement.jpg` with `https://example.com/video-poster.jpg` so it's shorter and easier to read.